### DISCUSSION TITLE
How to customize ForgotPassword

### DISCUSSION THREAD

**Discussion by go2ismail** [serenity-is/Serenity#7166] [Help] [answered]
Dear all,

I'm in process customizing membership page for re-branding / theme customization. Login and SignUp have no problem because the source code is already there. The problem is when to customize ForgotPassword because the source code is on Serenity Extension. 

The idea is to move the ForgotPassword from Serenity Extention to my local project and hopefully that will override the ForgotPassword from Serenity Extension, unfortunately still not success.

Are there anyone have the same experience regarding this?

Below are capture of my ForgotPassword structure:

![image](https://github.com/serenity-is/Serenity/assets/2275051/8173dc68-5536-4ca2-9f2b-0e1d2db26624)

Thanks in advance..

**Comment by VictorTomaili** [comment]
Override the related functions in `AccountPasswordActionsPage.cs` file.

**Comment by go2ismail** [comment, marked as answer]
Hi @VictorTomaili ,

Thank you for your suggestion, its work perfectly! 
I'm override the module folder path to target my customization:
![image](https://github.com/serenity-is/Serenity/assets/2275051/9dc66e9d-1156-4c8a-9bdb-071f1c80763f)

and everything working as expected:
![image](https://github.com/serenity-is/Serenity/assets/2275051/31edd0fd-58d4-4bc4-8491-c52dff91fa3d)
![image](https://github.com/serenity-is/Serenity/assets/2275051/4cea650d-53a3-43ed-89bf-5f79887bc636)

Thank you!

**Comment by tmartinscopo** [comment]
Hi @go2ismail ,
I'm very curious about the image of the Dashboard Desginer, it looks very good.  Is that devexpress? Is it too much to ask for a small code example of how you did this integration? 
It would also be interesting to see a small example of integration with a devexpress component/widget. 
If you can share it with the community, my hat's off to you. :)

Thanks

**Comment by go2ismail** [comment]
Hi @tmartinscopo ,

Sure, since it uses a commercial third-party library, I'm afraid you will need to purchase it:

[Removed - Ads are not allowed]

**Reply by teafmartins** [reply]
Hi, @go2ismail , 
Can you paste the code without any ad? :) it looks like it was removed by @VictorTomaili .
Thanks

**Reply by VictorTomaili** [reply]
It looks like he is posting an ad for their product. There is no code.

**Comment by go2ismail** [comment]
Hi @VictorTomaili ,
I'm sorry, I did not aware that ads is prohibited.

Hi @teafmartins,
Serenity is ASP.NET Core, you can consult DevExpress official docs about how to integrate their control with ASP.NET Core app.

**Comment by elenaaralla** [comment]
Hi all,

What if I would only overwrite the section _Title_ of ForgotPassword form?

Exists a file like AccountPanelTitle.tsx that I can overwrite?
